src/panel/details.tsx

```diff
@@ -195,19 +979,66 @@ interface DetailsProps { result: Result, resultsFixed: string[], height: IObserv
                                 });
                                 if (stack.message?.text) {
                                     return <div key={key} className="svStack">
-                                        <div className="svStacksMessage">
+                                        <div
+                                            className="svStacksMessage"
+                                            ref={(element) => {
+                                                if (element && stack?.message?.text) {
+                                                    const messageText = stack.message.text;
+                                                    // Check for truncation after layout
+                                                    setTimeout(() => {
+                                                        if (element.scrollWidth > element.clientWidth) {
+                                                            element.setAttribute('title', messageText);
+                                                        } else {
+                                                            element.removeAttribute('title');
+                                                        }
+                                                    }, 0);
+                                                }
+                                            }}
+                                        >
                                             {stack?.message?.text}
                                         </div>
                                         <div className="svDetailsBody svDetailsCodeflowAndStacks">
                                             <List items={stackFrames} renderItem={renderStack} selection={selection} allowClear />
                                         </div>
                                     </div>;
                                 }
+                                return null; // Return null for stacks without message text
```

Comment:
Should we be filtering out the nulls? When does this happen and what is the user experience?